.github/workflows/pre-commit.yml

```diff
@@ -1,20 +0,0 @@
-# https://github.com/pre-commit/action
```

Comment:
Did you really mean to delete the non-E2E workflows?

Reply:
I'll assume it's only to remove noise while you're iterating in the new one.

Reply:
It was to reduce the computation effort, as I don't want all the workflows to be run for each commit